Gemfile.lock

```diff
@@ -58,40 +75,48 @@ GEM
       nokogiri (>= 1.5.9)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
+    marcel (0.3.3)
+      mimemagic (~> 0.3.2)
     method_source (0.9.2)
+    mimemagic (0.3.10)
```

Comment:
I had to run `bundle update mimemagic` as several older versions have been yanked